AllReadyApp/Web-App/AllReady/Areas/Admin/ViewModels/Validators/Task/TaskEditViewModelValidator.cs

```diff
@@ -53,22 +53,22 @@ public TaskEditViewModelValidator(IMediator mediator)
             }
             
             // Rule - Attachments are optional
-            if (viewModel.NewAttachment != null && !string.IsNullOrEmpty(viewModel.NewAttachment.Name))
+            if (viewModel.NewAttachment != null && !string.IsNullOrEmpty(viewModel.NewAttachment.FileName))
```

Comment:
please update to:
`if (!string.IsNullOrEmpty(viewModel.NewAttachment?.FileName))`